workshops/flappy_learning/README.md

```diff
@@ -48,6 +48,8 @@ var start = function(){
 
 Then we will create a neuron preceptor network. And we will set the population of the baby bird to learn to 50. You can tweak the network of course, but I recommending it as [2, [2], 1] or [4, [4], 1].
 
+_Neuron Preceptor is a network design of the brain of the Machine Learning, similar to human that have its brain cell that united to be a network that is called brain_
```

Comment:
This is highly ambiguous and still doesn't actually tell me what this is

Reply:
Sorry for the bad sentences and definitions. I think it was cool. I will revisioning it.

Reply:
please check it here #1599